package.json

```diff
@@ -72,7 +72,7 @@
 		"source-map-support": "^0.5.21",
 		"ts-loader": "^9.4.2",
 		"ts-node": "^10.9.1",
-		"typescript": "^5.0.4",
+		"typescript": "^5.4.5",
```

Comment:
Why was this change made in this PR? (also another change in this file)

Reply:
I didn't do it on purpose, I think npm did it automatically when I ran `npm i`.